apis/nucleus/src/components/listbox/hooks/useOnTheFlyModel.jsx

```diff
@@ -122,11 +127,16 @@ export default function useOnTheFlyModel({ app, fieldIdentifier, stateName, opti
     fieldName = fieldIdentifier;
   }
 
-  if (frequencyMode !== 'N' || histogram) {
-    const field = fieldIdentifier.qLibraryId ? fieldDef : fieldName;
-    listdef.frequencyMax = {
-      qValueExpression: `Max(AGGR(Count([${field}]), [${field}]))`,
-    };
+  if (frequencyMode !== 'P' && histogram) {
```

Comment:
Should we update isFrequencyMaxNeeded above from 
`const isFrequencyMaxNeeded = options.histogram || options.frequencyMode !== 'N';`
to
`const isFrequencyMaxNeeded = options.histogram && options.frequencyMode === 'V';`

and use it here?

Reply:
No it is in different scope!